recipes/adcc/meta.yaml

```diff
@@ -0,0 +1,80 @@
+{% set version = "0.15.10" %}
+
+package:
+  name: adcc
+  version: {{ version }}
+
+source:
+  url: https://github.com/adc-connect/adcc/archive/v{{ version }}.tar.gz 
+  sha256: 8cd0fc53a0e4862421ff45b27058a7bae7e54738eea3a57ef75d221908f980a3
+
+build:
+  number: 0
+  skip: true  # [win or py<37]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - python
+    - libtensorlight >=3.0.1
+    - pkgconfig
+    - wget
+  host:
+    - python
+    - setuptools
+    - pip
+    - libtensorlight >=3.0.1
+    - pybind11 >=2.6
+    # Testing:
+    - pytest
+    - pytest-cov
+    - pyyaml
+    - pandas >=0.25.0
+    # Standard dependencies:
+    - numpy >=1.14
+    - h5py >=2.9
+    - scipy >=1.2
+    - tqdm >=4.30
+    - opt_einsum >=3.0
```

Comment:
I don't think any of these are used during package build.
```suggestion
```

Reply:
The test suite needs to be run in `build.sh`, that's why I put these dependencies in there.

Reply:
Done.